### ISSUE TITLE
convert to other python objects

### ISSUE THREAD

**Issue by mheskett** [daler/pybedtools#258]
Can you convert a pybedtools object to something like a list of lists, numpy ndarary or pandas data frame for custom analysis? 

thank you!

**Comment by daler** [comment]
Hi @mheskett, you can use the `BedTool.to_dataframe()` method to convert to a pandas dataframe, work on it in pandas, and then bring it back to pybedtools with `BedTool.from_dataframe()`.

**Comment by mheskett** [comment]
That is excellent, thank you!

**Comment by mheskett** [comment]
to_dataframe() names argument puts a default header as row 1.

if names are given then this should remove the 0th row here.

          chrom      start        end            name     length    strand
thickStart thickEnd  itemRgb blockCount blockSizes blockStarts        test
     test2
0    #1_usercol  2_usercol  3_usercol       4_usercol  5_usercol  6_pct_at
 7_pct_gc  8_num_A  9_num_C   10_num_G   11_num_T    12_num_N  13_num_oth
14_seq_len
1             1   55904048   56483852        test     579803  0.597914
 0.402086   177666   119190     113941     169007           0           0
    579804

**Comment by mheskett** [comment]
Okay thank you for the response, and my apologies for being unclear.
Bedtools Nuc returns a bed like file that has column headers--and this is
what should become the pandas column names. No--I am not using a non-bed
file with a header.
The below code is not ideal--but it would ignore the header returned
bedtools nuc, and then you would have to hardcode in the column names.

```
a.nucleotide_content(fi="hg38.nochr.fa")\
.to_dataframe(comment="#",names=['chrom', 'start', 'end', 'name', 'length',
'strand',
'thickStart', 'thickEnd', 'itemRgb', 'blockCount', 'blockSizes',
'blockStarts',"test","test2"]))
```
This will return a data frame with the bedtools nuc column names as the 0th
row which is also not ideal.
```
(a.nucleotide_content(fi="/Users/mike/replication_tcga/hg38.nochr.fa")
.to_dataframe()
```

**Comment by daler** [comment]
Addressed in #264; you can use

```python
a.nucleotide_content(fi='x.fa').to_dataframe(disable_auto_names=True)
```

(see test [here](https://github.com/daler/pybedtools/pull/264/commits/237583828c58b83463802d82a1c4f7f178ab8c5a#diff-e9ac3fd4ff0a367972edda9d92e0c999R2179))
